packages/paymaster/src/BiconomyPaymaster.ts

```diff
@@ -327,19 +331,24 @@ export class BiconomyPaymaster implements IHybridPaymaster<SponsorUserOperationD
         const preVerificationGas = response.result.preVerificationGas;
         const verificationGasLimit = response.result.verificationGasLimit;
         const callGasLimit = response.result.callGasLimit;
+        const maxPriorityFeePerGas = response.result.maxPriorityFeePerGas;
```

Comment:
I don't think now paymaster service returns the maxFeePerGas or maxPriorityFeePerGas data.